### PR TITLE
invite + minor change in README

### PR DESCRIPTION
**Pull request by layik**
Hi @Robinlovelace :)

The admin staff at LIDA need something like the invite doc I have added. Once you are happy I can send it to them. They are looking at booking a space as I do this.

Thanks!

### CODE REVIEW

File: invite.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+Event: LIDA Visualisation Hackathon 2024
+
+Title: Visualisation of Transport Network Hackathon
+
+Organiser: LIDA Visualisation Programme
+
+Date: Thursday 19th September 2024
```
Comment (Robinlovelace) [inline invite.md:7]:
```suggestion
Date: Thursday 3rd October 2024
```

Planning to be in Prague on 19th September, can we move it back a couple of weeks?

**Review by Robinlovelace — CHANGES_REQUESTED** [review]
Many thanks Layik, looking good, but can we change the date?

File: invite.md (hunk 1 above)
Comment (layik) [reply invite.md:7]: Absolutely, it seems room booking is not possible for the next year anyways, I better communicate this and maybe leave the room booking for start of Sep?

**Comment by Robinlovelace** [comment]
Start of October could work well. Thanks Layik!

**Pull request merged**